src/pae/ieee802_1x_kay.c

```diff
@@ -4209,7 +4210,7 @@ int ieee802_1x_kay_get_macsec(struct ieee802_1x_kay *kay, char *buf,
 			dl_list_for_each(txsa, &p->txsc->sa_list,
 					 struct transmit_sa, list) {
 				res = os_snprintf(pos2, end - pos2,
-						  "\t\tAN: %u\tActive: %s\tPN: %lu\n",
+						  "\t\tAN: %u\tActive: %s\tPN: %" PRIu64 "\n",
```

Comment:
`%zu` for size_t

Reply:
I see the problematic structure fields being explicity defined as u64 as show below.
Thus I think PRIu64 is the correct format specifier in this case.

/* TransmitSA in IEEE Std 802.1AE-2006, Figure 10-6 */
struct transmit_sa {
	bool in_use; /* bool inUse (read only) */
	**u64 next_pn**; /* PN nextPN (read only) */

/* ReceiveSA in IEEE Std 802.1AE-2006, Figure 10-6 */
struct receive_sa {
	bool in_use; /* bool inUse (read only) */
	**u64 next_pn**; /* PN nextPN (read only) */